src/components/dashboard/FaceVerification/hooks/useDisposingState.js

```diff
@@ -0,0 +1,55 @@
+import { useEffect, useRef, useState } from 'react'
```

Comment:
i think this whole component is an overkill
why not in IntroScreen just have an effect,[]?
```
[disposing,setDisposing] = useState(undefined)
const isDisposing = await api.isFaceSnapshotDisposing(enrollmentIdentifier).catch(e => undefined)
if(isDisposing) showdialog()
setDisposing(isDisposing)
```

Reply:
it works like that, just moved to the separate hook

Reply:
i've simplified a but hook, removed extra state vars like error, completed
now we have just disposing. if disposing===null -> request in progress, if true / false - request completed with this result